pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -217,3 +224,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
 | description | string        | Property description |
 | type        | string        | Property type        |
 | ?enum       | array(string) | Allowed enum values  |
+
+#### Metric Model
+
+| Attribute   | Type                  | Description                                                |
+| :---------- | :-------------------- | :--------------------------------------------------------- |
+| id          | string                | Metric id                                                  |
+| name        | string                | Metric name                                                |
+| description | string                | Metric description                                         |
+| categories  | array(string)         | Category ids where this metric is included                 |
+| type        | string                | One of Proportion / EventSegmentation / Funnel / Retention |
+| items       | array of Nested Event | Event descriptions                                         |
+
+#### Nested Event Model
+
+| Attribute | Type       | Description |
+| :-------- | :--------- | :---------- |
+| eventId   | string     | Event id    |
+| where     | array(???) | ???         |
+| groupBy   | array(???) | ???         |
```

Comment:
```suggestion
| groupBy   | array(groupByFilter)| array of objects containing property ids to group by |
```
##### groupByFilter
| Attribute      | Type       | Description                                              |
| :----------- | :-------  | :-----------------------------------  |
| propertyId   | string     | property Id being used in group filter   |